zhaquirks/tuya/ts0601_switch.py

```diff
@@ -312,6 +312,51 @@ class TuyaDoubleSwitchGP(TuyaSwitch):
     }
 
 
+class TuyaDoubleSwitchNOC(TuyaSwitch):
+    """Tuya double channel switch with No Output Clusters."""
+
+    signature = {
+        MODELS_INFO: [
+            ("_TZE200_wvovwe9h", "TS0601"),  # reported in #4077
+        ],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    TuyaOnOffManufCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT,
```

Comment:
Since this isn't changed, we don't need to include this in the `replacement`, or we can include both for clarity.
But just including one (when it's not unchanged) is a bit weird.

I'll also add the `PROFILE_ID` per the suggestion below. We generally include both in a lot of v1 quirks:
```suggestion
                PROFILE_ID: zha.PROFILE_ID,
                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT,
```